src/compiler/builders.js

```diff
@@ -21,7 +21,7 @@ const versionedBuilderMap = {
  */
 function wrap(buildDom) {
   return function wrapper(element) {
-    applyStaticLayout(element);
+    applyStaticLayout(/** @type {AmpElement} */ (element));
```

Comment:
@samouri This is a "new" type error. When we use the real `#core` types instead of your stubs, it comes up that this has to be an AMP element. That will obv not always be the case, ideally. Casting for now, but currently this would mask bugs with non-AMP elements.

Reply:
This should only function on AMP or Bento elements. Can tweak in the future.

Reply:
Actually, I think applyStaticLayout could work on any Element

Reply:
TODO(you)